src/main/java/com/checkmarx/jenkins/CxScanBuilder.java

```diff
@@ -1280,7 +1303,14 @@ private String getCxOriginUrl(EnvVars env, CxLoggerAdapter log) {
         }
         return originUrl;
     }
-
+    private Boolean verifyCustomCharacters(String inputString) {
+    	 Pattern pattern = Pattern.compile("(^([a-zA-Z0-9#._]*):([a-zA-Z0-9#._]*)+(,([a-zA-Z0-9#._]*):([a-zA-Z0-9#._]*)+)*$)");
+         Matcher match = pattern.matcher(inputString);
```

Comment:
What if inputString is null here @ENCHALWARPRANJAL ?
`java.lang.NullPointerException
 	at java.util.regex.Matcher.getTextLength(Matcher.java:1283)
 	at java.util.regex.Matcher.reset(Matcher.java:309)
 	at java.util.regex.Matcher.<init>(Matcher.java:229)
 	at java.util.regex.Pattern.matcher(Pattern.java:1093)
 	at com.checkmarx.jenkins.CxScanBuilder.verifyCustomCharacters(CxScanBuilder.java:1308)
 	at com.checkmarx.jenkins.CxScanBuilder.resolveConfiguration(CxScanBuilder.java:1338)
 	at com.checkmarx.jenkins.CxScanBuilder.perform(CxScanBuilder.java:902)
 	at jenkins.tasks.SimpleBuildStep.perform(SimpleBuildStep.java:123)
 	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:99)
 	at org.jenkinsci.plugins.workflow.steps.CoreStep$Execution.run(CoreStep.java:69)
 	at org.jenkinsci.plugins.workflow.steps.SynchronousNonBlockingStepExecution.lambda$start$0(SynchronousNonBlockingStepExecution.java:47)
 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
 	at java.lang.Thread.run(Thread.java:748)
 Finished: FAILURE`

Reply:
Hi [joevin-slq-ibp](https://github.com/joevin-slq-ibp) can you mention the steps to reproduce this issue.

Reply:
@ENCHALWARPRANJAL : i suggest to talk with Yair from the EMEA support team, I've created the case [#143029](https://checkmarx.force.com/CheckmarxCustomerServiceCommunity/s/case/5004K00000NIGAh/) with a Jenkinsfile to reproduce it.